docs/contributing/getting_ready.rst

```diff
@@ -0,0 +1,300 @@
+        start, if Windows is the only operating system you use, or if
+        you have not used Unix before.
+
+      * We recommend `Windows Subsystem for Linux`_ (WSL) if you are
+        familiar with Unix, you use macOS or Linux too, or you expect to
+        contribute to PlasmaPy extensively. These instructions cover
+        `installing WSL`_. If you choose WSL, please follow the tabs for
+        :guilabel:`Linux/WSL` in the following sections.
+
+   .. group-tab:: macOS
+
+      In the :guilabel:`Finder`, go to :guilabel:`Applications`. Enter
+      the :guilabel:`Utilities` folder and double click on
+      :guilablel:`Terminal`.
+
+   .. group-tab:: Linux/WSL
+
+      Open a terminal by using :kbd:`Ctrl + Alt + T`.
+
+Installing Python
+-----------------
+
+There multiple ways to install Python. PlasmaPy requires a version of
+Python between |minpython| and |maxpython|. We recommend using Python
+|maxpython|.
+
+We recommend using Anaconda_, which is widely used as a package
+management system and environment management system in the data science
+and scientific Python communities. Anaconda includes `Anaconda
+Navigator`_ as its graphical user interface (GUI) and Conda_ as its
+command line interface (CLI). Anaconda allows us to install different
+versions of Python into different Python, environments and can also be
+used to distribute packages in languages besides Python.
+
+* If you prefer a graphical user interface, please following these
+  instructions on `installing Anaconda Navigator`_.
+* If you prefer using the command line, please follow these instructions
+  on `installing Conda`_.
+
+Anaconda is not the only good way to install Python. Python's website
+describes how to `download Python`_, and `Real Python`_ has instructions
+on `installing Python`_ for several different operating systems (if
+using WSL_, follow the Linux instructions).
+
+Using git and GitHub
+--------------------
+
+Plasma code development is done using |git|_ and GitHub_. Before
+contributing code to PlasmaPy:
+
+#. `Sign up on GitHub`_ for a free account.
+
+#. Verify that |git|_ is installed. Open a terminal and run:
+
+   .. code-block:: bash
+
+      git --version
+
+   If there is a ``command not found`` error, then please follow these
+   instructions to `install git`_.
+
+#. Optionally configure |git|_ with your name and email using the
+   following commands in the terminal, where the name and email are
+   replaced with your own.
+
+   .. code-block:: bash
+
+      git config --global user.name "Your Name"
+      git config --global user.email "your.email@example.com"
+
+   You may also set your default editor with a command like one of the
+   following:
+
+   .. code-block:: bash
+
+      git config --global core.editor emacs
+      git config --global core.editor notepad
+
+   For more editor options, see this page on `git commands for setup and
+   config`_.
+
+#. `Add a new SSH key to your GitHub account`_.
+
+Initial setup
```

Comment:
A cool idea could be to integrate short recorded gifs of the steps you reference to visually show the step for further clarity. It could also make the page feel more alive if there are more visuals.

Reply:
If we're going in that direction, I'll point out the existence of asciinema; and then we could use https://github.com/asciinema/agg to turn that into gifs, even!